src/cc_toolchain/cc_toolchain_config.bzl

```diff
@@ -109,7 +109,6 @@ def _make_common_features(ctx):
 
     result["static_link_cpp_runtimes"] = feature(
         name = "static_link_cpp_runtimes",
```

Comment:
Nit: 


```suggestion
        name = "static_link_cpp_runtimes"
```

Reply:
That might not work with bazel's formatting, it tends to add commas like this to minimize diffs. Worth a try but may get auto-formatted.

Reply:
That won't pass formatting I'm afraid. In any case I strongly prefer this style because it makes things easy to move around later without hurting clarity.